lua/nvim-highlight-colors/utils.lua

```diff
@@ -36,6 +36,13 @@ end
 ---@param color_value string
 ---@return string
 function M.create_highlight_name(color_value)
+	-- Checking to match on ansi codes to remove naughty chars
+	if string.match(color_value, "\\") then
+		local g1, g2 = string.match(color_value, "([0-9]);?([0-9]+)m")
+		if g1 and g2 then
+			color_value = g1 .. g2
+		end
+	end
```

Comment:
Could you remove `naughty chars` using the gsub below? `[!(),%s%.-/%%=:\"']+`

Reply:
```
	return 'nvim-highlight-colors-' .. string.gsub(color_value, "#", ""):gsub("\\[0-9]*%[", ""):gsub("[!(),%s%.-/%%=:\"';%[%]]+", "")
	```
	
let me know if this is what you were after, i believe this should be more ideal